docs/sphinx/using/examples/multi_gpu_workflows.rst

```diff
@@ -151,5 +139,65 @@ Now let's try to time multi GPU run.
 
 .. parsed-literal::
 
-    85.3 ms ± 2.36 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)
-
+    1.1754660606384277
+
+
+    
+Multi-QPU + Other Backends (`remote-mqpu`)
+-------------------------------------------
+    
+    
+The `mqpu` backend can be extended so that each parallel simulated QPU can be simulated with backends other than :code:`nvidia`.  This provides a way to simulate larger scale circuits and execute parallel algorithms. This accomplished by launching remotes servers which each simulated a QPU.  
```

Comment:
```suggestion
The `mqpu` backend can be extended so that each parallel simulated QPU run backends other than :code:`nvidia`.  This provides a way to simulate larger scale circuits and execute parallel algorithms. This accomplished by launching remotes servers which each simulated a QPU.  
```